Makefile

```
@@ -13,7 +13,7 @@ clean:
 install:
 	go mod download && go mod verify
 	go install ./...
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.47.3
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.2
```

Less wiggle room here, we should stay consistent with `go install ./...`
```suggestion
	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
```